lib/features/editor-actions/BpmnEditorActions.js

```diff
@@ -174,4 +175,10 @@ BpmnEditorActions.prototype._registerDefaultActions = function(injector) {
     });
   }
 
-};
\ No newline at end of file
+  if (eventBus) {
+    this._registerAction('saveEditing', function() {
```

Comment:
Can't we directly use `directEditing#complete` here? Is there a reason why we have to introduce a dedicated event for this?

Reply:
Because the completion if direct editing is taken care by LabelEditingProvider class [here](https://github.com/bpmn-io/bpmn-js/blob/44ba918d7e823001e1e2f7aebff44022ae518f5e/lib/features/label-editing/LabelEditingProvider.js#L55), rather than duplicating the direct editing completion logic, I wanted LabelEditingProvider to take care of this case also